samples/health-check/liveness-command.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2017 Istio Authors
```

Comment:
2018?

Reply:
Good catch, but it's already submitted :-)